client/state/login/reducer.js

```diff
@@ -370,10 +370,10 @@ export const socialAccount = ( state = { createError: null }, action ) => {
 	return state;
 };
 
-const userExistsErrorHandler = ( state, { error, authInfo } ) => {
-	if ( error.code === 'user_exists' ) {
+const socialResponseErrorHandler = ( state, { error, authInfo } ) => {
+	if ( error.code === 'user_exists' || error.code === 'unknown_user' ) {
```

Comment:
Need to look into this change.

Reply:
> Need to look into this change.

Ah, it was to set additional `store` state in 5532501c42629a5c34dd1d76679fc7677250f67f

Because I reverted that code in e5a8afd9c55b3b27bfc12588a3182ef9ed17979e, its safe to revert the remaining d2b05caa4b3c678807fe5c5538909d6d8f42ea90